internal/infrastructure/mongo/mongodoc/asset_test.go

```diff
@@ -0,0 +1,147 @@
+package mongodoc
+
+import (
+	"testing"
+	"time"
+
+	"github.com/reearth/reearth-backend/pkg/asset"
+	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func TestNewAsset(t *testing.T) {
+	type args struct {
+		asset *asset.Asset
+	}
+
+	newAsset := asset.New().
+		NewID().
+		Team(id.NewTeamID()).
+		Name("test").
+		Size(10).
+		URL("test_url").
+		MustBuild()
+
+	tests := []struct {
+		name  string
+		args  args
+		want  *AssetDocument
+		want1 string
+	}{
+		{
+			name: "new asset",
+			args: args{
+				asset: newAsset,
+			},
+			want: &AssetDocument{
+				ID:          newAsset.ID().String(),
+				CreatedAt:   newAsset.CreatedAt(),
+				Team:        newAsset.Team().String(),
+				Name:        newAsset.Name(),
+				Size:        newAsset.Size(),
+				URL:         newAsset.URL(),
+				ContentType: newAsset.ContentType(),
```

Comment:
```suggestion
	assetID := id.NewAssetID()
	teamID := id.NewTeamID()
	now := time.Now()

	tests := []struct {
		name  string
		args  args
		want  *AssetDocument
		want1 string
	}{
		{
			name: "new asset",
			args: args{
				asset: asset.New().
				    ID(assetID).
				    Team(teamID).
				    CreatedAt(now).
				    Name("test").
				    Size(10).
				    URL("test_url").
				    ContentType("application/json").
				    MustBuild(),
			},
			want: &AssetDocument{
				ID:          assetID.String(),
				CreatedAt:   now,
				Team:        teamID.String(),
				Name:        "test",
				Size:        10,
				URL:         "test_url",
				ContentType: "application/json",
```

It is easier to increase the number of test cases by generating IDs in advance and using them rather than using the Asset itself.